### PR TITLE
[FrameworkBundle] Tidied up ternary operators

### PR DESCRIPTION
**Pull request by SidRoberts**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | none
| License       | MIT
| Doc PR        | none

Here were some instances where the same method or expression was being called more than necessary.

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
Even if the new code doesn't behave exactly as the current one, I think we don't care in this specific case.
This change should be done against the lowest branch where it applies - 2.7 here I guess?

File: src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php [hunk 1]
```diff
@@ -85,7 +85,7 @@ protected function describeRoute(Route $route, array $options = array())
             array('Route Name', isset($options['name']) ? $options['name'] : ''),
             array('Path', $route->getPath()),
             array('Path Regex', $route->compile()->getRegex()),
-            array('Host', ('' !== $route->getHost() ? $route->getHost() : 'ANY')),
+            array('Host', ($route->getHost() ?: 'ANY')),
```
Comment (ro0NL) [inline src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php]: remove outer brackets?

**Comment by Tobion** [comment]
This breaks when the value is `'0'` and is inconsistent with the component logic. So I'm 👎

**Comment by fabpot** [comment]
Closing then.

**Pull request closed without merging**